include/SFML/Graphics/RenderTexture.hpp

```diff
@@ -72,11 +72,15 @@ class SFML_GRAPHICS_API RenderTexture : public RenderTarget
     /// Before calling this function, the render-texture is in
     /// an invalid state, thus it is mandatory to call it before
     /// doing anything with the render-texture.
+    ///
     /// The last parameter, \a depthBuffer, is useful if you want
     /// to use the render-texture for 3D OpenGL rendering that requires
     /// a depth buffer. Otherwise it is unnecessary, and you should
     /// leave this parameter to false (which is its default value).
     ///
+    /// After creation, the contents of the render-texture is undefined, call
+    /// RenderTexture::clear first to ensure a single color fill
```

Comment:
This would be clearer in two sentences IMO.
Besides, a final period is missing.

Reply:
Adjusted it accordingly